src/com/untamedears/PrisonPearl/CombatTagListener.java

```diff
@@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
         if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
             return;
         }
-        UUID plruuid = event.getPlayerUUID();
+        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
```

Comment:
Probably not then I was having problems getting that to compile so I just
pulled one off Jenkins

On Friday, February 27, 2015, rourke750 <notifications@github.com> wrote:

> In src/com/untamedears/PrisonPearl/CombatTagListener.java
> <https://github.com/Civcraft/PrisonPearl/pull/34#discussion_r25510303>:
>
> > @@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
> >          if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
> >              return;
> >          }
> > -        UUID plruuid = event.getPlayerUUID();
> > +        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
>
> Ah I see what you mean. Are you running the lastest combat tag?
> getPlayerUUID is in the most current source
> https://github.com/cheddar262/Combat-Tag/blob/master/src/main/java/com/trc202/CombatTagEvents/NpcDespawnEvent.java
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Civcraft/PrisonPearl/pull/34/files#r25510303>.
>